src/client/components/pages/entity-revisions.js

```diff
@@ -16,6 +16,7 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+
```

Comment:
Shouldn't need to add this line at all - it's just left over from the eslint comment you added earlier.